data/sources/nz-cruise-association/nz-cruise-schedule.csv

```diff
@@ -0,0 +1,1124 @@
+Ship,Port,Arrival,Arrival time,Departure,Departure time,Guests (Lower Berths),Crew
```

Comment:
Different column naming style to the other file

Reply:
@whistlebee good catch, shallChangeToCamelCase 😆

Reply:
Sounds good, agree to keeping the source version intact